packages/workbox-google-analytics/lib/constants.mjs

```diff
@@ -0,0 +1,23 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import '../_version.mjs';
+
+export const CACHE_NAME = 'workbox-google-analytics';
```

Comment:
Is it worth moving this to workbox-core just to make sure it follows the same pattern as runtime and precaching?

Reply:
Hmmm, yeah probably.